admin/src/components/ImportModal/ImportModal.js

```diff
@@ -53,7 +53,7 @@ export const ImportModal = ({ onClose }) => {
   };
 
   const readFile = (file) => {
-    if (file.type === 'text/csv') {
+    if (file.type === 'text/csv' || /\.csv$/i.test(file.name)) {
       setDataFormat(dataFormats.CSV);
     } else if (file.type === 'application/json') {
```

Comment:
Equivalent check can be done for json files

Reply:
done with it, my previous code is really rough, I'm learning strapi.

Reply:
It looks good 👍 your feature is merged